src/main/resources/db/migration/V0005__Insert_test_data_animals.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `animals` VALUES (1,'Black','Super Dog',2,'2023-08-22 07:18:11','2023-08-22 07:18:11'),
```

Comment:
Why we divided sql queries to more one file ?

Reply:
I guess that for migration and develop DB must be more than  one file.Step by stap. If I am wrong, I will make one file.

Reply:
done